spec/requests/dashboard_spec.rb

```diff
@@ -80,8 +80,16 @@
         expect(response.body).not_to include("Pro Analytics for #{organization.name}")
       end
 
-      it "does not render a link to upload a video" do
+      it "does not render a link to upload a video when enable_video_upload is false" do
         get dashboard_path
+        SiteConfig.enable_video_upload = false
```

Comment:
We should use `allow(SiteConfig).to receive(:enable_video_upload).and_return(false)` so this doesn't stay set for other tests

Reply:
✅